frontend/src/metabase/questions/questions.js

```diff
@@ -176,7 +176,7 @@ export const setCollection = createThunkAction(SET_COLLECTION, (cardId, collecti
                 )));
                 MetabaseAnalytics.trackEvent("Questions", "Bulk Move to Collection");
             }
-            selected.map(item => dispatch(setCollection(item.id, collectionId)));
+            selected.map(item => dispatch(setCollection(item.id, { id: collectionId })));
```

Comment:
so the idea is that you can move it into a collection by either the id or the slug?